modules/core/src/main/scala/io/janstenpickle/trace4cats/Span.scala

```diff
@@ -156,6 +156,8 @@ object Span {
 
   def noop[F[_]: Applicative]: Resource[F, Span[F]] = Resource.pure[F, Span[F]](NoopSpan[F](SpanContext.invalid))
 
+  def noopInstance[F[_]: Applicative]: Span[F] = NoopSpan[F](SpanContext.invalid)
```

Comment:
Before 0.12, one could create `NoopSpan` directly, very useful for tests. Now that the constructor is private, I smuggled this little helper in.